IM_calculation/test/waveforms/moderate_magnitude/SEDS.000

```diff
@@ -3775,4 +3775,3 @@ SEDS       000
  -1.36923e-04  -1.36927e-04  -1.36930e-04  -1.36933e-04  -1.36939e-04  -1.36945e-04
  -1.36952e-04  -1.36960e-04  -1.36969e-04  -1.36977e-04  -1.36983e-04  -1.36989e-04
  -1.36992e-04  -1.36994e-04  -1.36994e-04  -1.36993e-04  -1.36992e-04  -1.36991e-04
-
```

Comment:
I presume that moving these files doesn't break the adv_im tests? It appears to pass.

Reply:
there are no adv_im tests, only a shell script with paths given

Reply:
The adv im tests are done manually, as required when anything adv IMs/OpenSees related is changed. The users involved are fine with the path change